pull/src/handler.cc

```diff
@@ -113,7 +113,7 @@ static std::size_t WriteResponse(struct mg_connection* conn,
   return body.size();
 }
 
-bool MetricsHandler::handleGet(CivetServer* server,
+bool MetricsHandler::handleGet(CivetServer* /*server*/,
```

Comment:
you could also write `std::ignore = server;` in the body. I always find the commented out parameters a liite bit clumsy.

Reply:
I followed the way clang-tidy does it:
https://clang.llvm.org/extra/clang-tidy/checks/misc-unused-parameters.html

So if /* ... */ is too clumsy isn't it then better to omit the name altogether?

I now removed the unused parameter name but I can change it to std::ignore if you think this is better.